doc/faq.md

```diff
@@ -20,12 +20,12 @@ If this line is present but the value is "false", then the org admin will need t
 
 After the feature is turned on, the reuse works automatically unless options like "--ignore-reuse" is specified at the runtime.
 
-Q: Is there a way to configure dx workflows so that only the files designated as workflow outputs are kept upon completion? Or perhaps a way to move intermediate files into a different output directory?
+> Q: Is there a way to configure dx workflows so that only the files designated as workflow outputs are kept upon completion? Or perhaps a way to move intermediate files into a different output directory?
 
 A: The intermediate output files can be moved to a specified folder by using the "--reorg" option during runtime. Detailed information on how to use it can be found in
 
 [https://github.com/dnanexus/dxWDL/blob/master/doc/ExpertOptions.md#handling-intermediate-workflow-outputs](https://github.com/dnanexus/dxWDL/blob/master/doc/ExpertOptions.md#handling-intermediate-workflow-outputs)
 
-Q: What does dxFUSE do if you try to randomly access parts of files that are streamed? As far as I know, you can do this with the regular FUSE package.
+> Q: What does dxFUSE do if you try to randomly access parts of files that are streamed? As far as I know, you can do this with the regular FUSE package.
 
 A: dxFUSE random access performance is good enough if you are accessing a small part of a file. Otherwise you are better of downloading the file to the local storage insstead of streaming. dxFUSE support mounting Dx containers to access its contents.
```

Comment:
minor typos, 
`better of` -> `better off`
`insstead` -> `instead`
`dxFuse support` `dxFuse supports`